### PR TITLE
chore(docs): update the secrets flag for task run

### PR DESCRIPTION
**Pull request by KollaAdithya**
address part of https://github.com/aws/copilot-cli/issues/4989#issuecomment-1604101419

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/docs/commands/task-run.en.md [hunk 1]
```diff
@@ -63,7 +63,8 @@ Task Configuration Flags
       --platform-os string             Optional. Operating system of the task. Must be specified along with 'platform-arch'.
       --resource-tags stringToString   Optional. Labels with a key and value separated by commas.
                                        Allows you to categorize resources. (default [])
-      --secrets stringToString         Optional. Secrets to inject into the container. Specified by key=value separated by commas. (default [])
+      --secrets stringToString         Optional. Secrets to inject into the container. Specified by key=value separated by commas. (default []). 
+                                       For secrets stored in AWS Parameter Store you can either specify name or ARN. For the secrets stored in AWS Secrets Manager you need to specify ARN.
```
Comment (dannyrandall) [inline site/content/docs/commands/task-run.en.md]: to keep lines <80 width
```suggestion
                                       For secrets stored in AWS Parameter Store you can either specify name or ARN.
                                       For the secrets stored in AWS Secrets Manager you need to specify ARN.
```

File: site/content/docs/commands/task-run.en.md [hunk 2]
```diff
@@ -116,4 +117,9 @@ $ copilot task run --platform-os WINDOWS_SERVER_2019_CORE --platform-arch X86_64
 Run a Windows 2022 task with the minimum cpu and memory values.
 ```console
 $ copilot task run --platform-os WINDOWS_SERVER_2022_CORE --platform-arch X86_64 --cpu 1024 --memory 2048
+```
+
+Run a task with secrets from AWS Secrets Manager injected into the container.
```
Comment (dannyrandall) [inline site/content/docs/commands/task-run.en.md]: nit
```suggestion
Run a task with a secret from AWS Secrets Manager injected into the container.
```

**Review by dannyrandall — APPROVED** [review]
lgtm, just some small things!

File: site/content/docs/commands/task-run.en.md (hunk 1 above)
Comment (Lou1415926) [reply site/content/docs/commands/task-run.en.md]: nit: should it be "...the names or the ARNs" and "you need to specify the ARNs"? @dannyrandall @huanjani @efekarakus english experts 🥺

**Review by Lou1415926 — APPROVED** [review]

File: site/content/docs/commands/task-run.en.md (hunk 1 above)
Comment (dannyrandall) [reply site/content/docs/commands/task-run.en.md]: i think the plurals you suggested make more sense!

**Pull request merged**